__tests__/index.tests.js

```diff
@@ -0,0 +1,26 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed');
+const topicData = require('../db/data/test-data/topics');
+const userData = require('../db/data/test-data/users');
+const articleData = require('../db/data/test-data/articles');
+const commentData = require('../db/data/test-data/comments');
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe('/api/topics', () => {
+  test('GET:200 sends an array of topics to the client', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics).toEqual(expect.any(Array));
```

Comment:
This test isn't too bad, but if we wanted to be a little more confident, we might check things like the length of the array and the shape of the objects inside rather than just the first object. Try changing this test to do those two things.